network/metadata.csv

```diff
@@ -73,14 +73,34 @@ system.net.tcp.send_q.avg,gauge,,byte,,The average TCP send queue size.,0,system
 system.net.tcp.send_q.count,rate,,connection,second,The rate of connections.,0,system,send-q count
 system.net.tcp.send_q.max,gauge,,byte,,The maximum TCP send queue size.,0,system,send-q max
 system.net.tcp.send_q.median,gauge,,byte,,The median TCP send queue size.,0,system,send-q median
+system.net.tcp4.close,gauge,,connection,,The number of closed TCP IPv4 connections.,0,system,tcp4 close
+system.net.tcp4.close_wait,gauge,,connection,,The number of TCP IPv4 connections waiting for termination.,0,system,tcp4 close_wait
 system.net.tcp4.closing,gauge,,connection,second,The number of TCP IPv4 closing connections. ,0,system,tcp4 closing
+system.net.tcp4.estab,gauge,,connection,,The number of open TCP IPv4 connections.,0,system,tcp4 estab
 system.net.tcp4.established,gauge,,connection,second,The number of TCP IPv4 established connections. ,0,system,tcp4 established
+system.net.tcp4.fin_wait_1,gauge,,connection,,The number of TCP IPv4 connections waiting for a connection termination request or an ack of a connection termination request sent previously.,0,system,tcp4 fin_wait_1
+system.net.tcp4.fin_wait_2,gauge,,connection,,The number of TCP IPv4 connections waiting for a connection termination request.,0,system,tcp4 fin_wait_2
+system.net.tcp4.listen,gauge,,connection,second,The number of TCP IPv4 listening connections.,0,system,tcp4 listen
```

Comment:
Should this be `connections` instead of  `connections/second`? Or did you use `connections/second` for consistency?

Reply:
Good catch, this is just a C&P error. Should be just `connections`, let me fix.